styles/docblock-python.less

```diff
@@ -6,3 +6,8 @@
 
 .docblock-python {
 }
+
+.current {
+  font-weight: bold;
```

Comment:
Could you make this class name a bit more specific?
I don't think it will interfere with future classes but you never know.